opensuse-base-image/src/main/docker/scripts/check-create-pgdb-impl.sh

```diff
@@ -171,4 +238,5 @@ EOF
 
 check_variables
 check_psql
+create_pgpass_file
```

Comment:
I originally had a cleanup function at the end of this script that removed the .pgpass file and unset the PGPASSFILE env var, but then I realised it wasn't getting executed because we exit the script in various places, both during a successful run and during failed runs, so it was getting a bit complicated to make sure the cleanup was called in all these scenarios, and then also handling the case where cleanup might also fail, so I dropped it:

https://github.com/CAFapi/opensuse-base-images/pull/20/commits/5dcea288c8344c202802bd9ac28c0ccdb90c4bfd#diff-f54a9b21898d4256e83beb7818f6e438247429236f2c658888d35f612d152cd3L238

But I can add it in again if you think it is important?

Reply:
Yes I think we probably should include the cleanup code.